core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -704,8 +704,11 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
                   }
                 );
 
+                // Changed from to unshift to address #1894, now focus
```

Comment:
This comment is cryptic: people looking at the code in the future will not care what the old value was. I suggest the following comment instead:

```
If the current category is not in the dropdown, add it as the first option.
```

and similarly below.